ssntp/server.go

```diff
@@ -246,6 +247,15 @@ func (server *Server) Serve(config *Config, ntf ServerNotifier) error {
 	transport := config.transport()
 	uri = config.URI
 
+	payload, err := configuration.GetConfiguration(config.ConfigURI)
+
+	if err != nil {
+		server.log.Errorf("Error loading configuration: %s", err)
+		return err
```

Comment:
Is it OK to **stop** the SSNTP server when no configuration is provided to start it for the first time?